viahtml/templates/viahtml/wrapper.html.jinja2

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>{% block title %}Via{% endblock %}</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700" rel="stylesheet" type="text/css">
+
+    <link rel="stylesheet" href="/static/viahtml/style.css">
+    <script async src="https://www.google-analytics.com/analytics.js"></script>
+    <script>
+        window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
+        ga('create', 'UA-26026798-1', 'auto');
+        ga('send', 'pageview');
```

Comment:
Does this cause all kinds of horrible because we don't have our own code for ViaHTML here? I don't know what we do with these metrics if anything.